playbooks/kind-integration-test-arm64/run.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-docker-ce
+    - andrewrothstein.bazel
+  vars:
+    - go_version: 1.12
+    - bazel_ver: '0.23.1'
+
+  tasks:
+    - name: git required repositories
+      shell:
+        cmd: |
+          go get sigs.k8s.io/kind
+          go get k8s.io/kubeadm
+          go get k8s.io/kubernetes
+          go get k8s.io/test-infra
+        executable: /bin/bash
+      environment: '{{ global_env }}'
+    - name: build KIND against k/k master using bazel
+      shell:
+        cmd: |
+          cd $GOPATH/src/sigs.k8s.io/kind && go install .
+          kind build base-image --image kindest/base:latest --source $GOPATH/src/sigs.k8s.io/kind/images/base --loglevel debug
+          kind build node-image --type bazel --image kindest/node:latest --base-image kindest/base:latest --kube-root=$GOPATH/src/k8s.io/kubernetes/ --loglevel debug
```

Comment:
I think we can just remove this one if you are using ``--build=bazel`` in the below command, it takes around 20 mins for each build in our vexxhost vm, so quite time consuming to build twice.